route/gin_test.go

```diff
@@ -39,6 +39,7 @@ func TestRun(t *testing.T) {
 			name: "error when default host is empty",
 			setup: func(host string) error {
 				mockConfig.On("GetString", "route.host").Return(host).Once()
+				mockConfig.On("GetString", "route.port").Return("test_ca.port").Once()
```

Comment:
We didn't need to add this, because we are testing `host can't be empty`.
We need to add a test case: `port can't be empty`.

Reply:
Yes I have added it.